easybuild/easyconfigs/k/krbalancing/krbalancing-0.5.0b0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
```

Comment:
Should be `PythonBundle`?

Reply:
I deliberately started using `PythonPackage` for this one, because it was getting a bit messy when using `exts_list` because I needed a pretty long `preinstallopts`.

I changed tactics to using a patch later though, so now it should indeed be fine to use `PythonBundle` instead, I'll change that.

Reply:
fixed in 01d58abec4